app/src/main/java/fr/iut_rodez/pathpilot_android_client/login/Login.java

```diff
@@ -0,0 +1,85 @@
+package fr.iut_rodez.pathpilot_android_client.login;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.util.Log;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import fr.iut_rodez.pathpilot_android_client.MainActivity;
+import fr.iut_rodez.pathpilot_android_client.R;
+import fr.iut_rodez.pathpilot_android_client.util.Network;
+import fr.iut_rodez.pathpilot_android_client.util.Popup;
+import fr.iut_rodez.pathpilot_android_client.util.ValidateForm;
+
+public class Login extends AppCompatActivity {
```

Comment:
Missing Javadoc comment to describe what the activity is for

Reply:
Fix in https://github.com/SAE-S5-01-Developpement-avance/PathPilot-android-client/pull/38/commits/3f381088a74825447a65980cc8bb98e00dd60954